### PR TITLE
change pinyin

### PR DESCRIPTION
**Pull request by chernochiu**
改变为拼音

### CODE REVIEW

File: hello.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-你好世界!
```
Comment (xmtq) [inline hello.txt:1]: sss

Comment (chernochiu) [reply hello.txt:1]: qqq

Comment (xmtq) [inline hello.txt:1]: pinyin + chinese

Comment (chernochiu) [reply hello.txt:1]: fine

Comment (chernochiu) [reply hello.txt:1]: > pinyin + chinese

fine

**Pull request merged**